models/settings/fields.yaml

```diff
@@ -53,3 +53,13 @@ tabs:
                     value:
                         type: text
                         label: Header
+        response_status_code:
+            label: Response status code
```

Comment:
Is it for all responses, or just one? I'd specify a little which response status code it includes.